src/components/participants/Participants.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { toast } from "react-toastify";
+import Input from "../form/Input";
+import { useState, useEffect } from "react";
+import Button from "../Button";
+import UserCard from "../user/UserCard";
+import { addParticipant, getUserData } from "../../comutils";
+
+const Participants = ({ id, participants, setLoading }) => {
```

Comment:
I think it's a little unnecessary this refactor because you are abstracting by abstracting, but it's ok. For the next time please make other PR to make these kind of changes.

Reply:
Reverted and solved